### ISSUE TITLE
wiki UX [unstable]

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/home#546]
What needs to be covered (if not covered already)

- [x] complete walkthroughs that can be used (loading, reloading, example wiki) done [here](https://github.com/threefoldtech/jumpscaleX_threebot/blob/unstable/docs/wikis/tech/quickstart.md)
- [x] ensuring good enough documentation on the current macros
- [x] links
- [x] searching (should work with changes)
- [x] update wiki of wiki_examples package so every document that uses `gdrive` links to [this](https://github.com/threefoldtech/jumpscaleX_threebot/tree/unstable/docs/wikis/tech#setting-up-gdrive-and-service-account) and [this](https://github.com/threefoldtech/jumpscaleX_threebot/blob/unstable/docs/wikis/tech/gdrive.md#how-to-configure).
- [x] navigation (clear instructions and notes on the navigation in wikis and need to be visual in the UI)
- [x]  update docs for wiki-reload web (with a screenshot).
- [x] don't require md extension https://github.com/threefoldtech/jumpscaleX_threebot/issues/376
- [x] wrong url in wiki view button https://github.com/threefoldtech/jumpscaleX_threebot/issues/377
- [x] grid, info_grid both docsites exist https://github.com/threefoldtech/jumpscaleX_threebot/issues/381
- [x] full reset doesn't clean the output docsify dir : https://github.com/threefoldtech/jumpscaleX_threebot/issues/380
- [x] fix web reload problems
- [x] links should be cases insensitive https://github.com/threefoldtech/jumpscaleX_threebot/issues/378

**Comment by abom** [comment]
For navigation, it's up to `_sidebar.md` and `docsfiy.js` configuration, as the side navigation bar is generated by `docsfiy.js`.

**Comment by waleedhammam** [comment]
**Branch: unstable**
**commits** https://github.com/threefoldtech/jumpscaleX_threebot/commit/84e8d6e8c44139fd68021cd064a667f6e1a75a2d, https://github.com/threefoldtech/jumpscaleX_threebot/commit/669375b2dddd9ed0e6772ec5144bb4e0b9c747d2, https://github.com/threefoldtech/jumpscaleX_threebot/commit/b61addf4378944403e44e61f75c3cf2741993842, https://github.com/threefoldtech/jumpscaleX_threebot/commit/54eeb36f606e820012cb3ae8a7e3ee7a9b57ee75
**changelog**
- Fixed test links, wiki web reload

- check at: http://165.22.206.215/wiki/testwikis#/test_include
- http://165.22.206.215/wiki/

**Comment by xmonader** [comment]
- notes on wiki pages requiring gdrive client in wiki_example threefoldtech/jumpscaleX_threebot@2203e8c20ef3087e97a94ace97444c15d84c693b
- update docs on sidebar, and reload threefoldtech/jumpscaleX_threebot@ec13c589bfd2339bcb02c40549f61cfa4fcfcc08

**Comment by waleedhammam** [comment]
Walkthrough here: https://github.com/threefoldtech/jumpscaleX_threebot/blob/88d24a060155eb16fdcf35818020dd1673c10be9/docs/wikis/tech/quickstart.md

**Comment by waleedhammam** [comment]
Fix reload blocked on: https://github.com/threefoldtech/jumpscaleX_core/issues/542

**Comment by gmachtel** [comment]
@waleedhammam Please recheck, https://github.com/threefoldtech/jumpscaleX_core/issues/542 has been delayed because no support for different BCDB processes and need to go through Gedis. So reason for blocking can't be right if it's still needed in 10.3 release.

**Comment by waleedhammam** [comment]
@gmachtel there 2 blocking issues, the links one.
the second `reload` which related to https://github.com/threefoldtech/jumpscaleX_core/issues/542 which is reload. it's working but with another way, we wanted it to use config manager but seems like it won't work.

**Comment by gmachtel** [comment]
@waleedhammam All I’m saying is that trying to solve an issue in release 10.3 where it’s depending on another issue that is only planned for 10.4 won’t work. Please remove dependency by either skipping these items in the issue or find another way.

**Comment by waleedhammam** [comment]
@gmachtel updated now

**Comment by Dina-Abd-Elrahman** [comment]
##verification steps##

- test load, reload and wikis example [here](https://github.com/threefoldtech/jumpscaleX_threebot/blob/unstable/docs/wikis)

error in wiki load from the server shell

![ff](https://user-images.githubusercontent.com/21218053/76499584-aad89e00-6447-11ea-894e-b46d3ddf9c4a.png)
 
can only load threebot wiki package from the server shell

-  searching is working fine 
![ff](https://user-images.githubusercontent.com/21218053/76499856-2c303080-6448-11ea-9db8-a60c1da0e2c6.png)

- making sure info_grid both docsites exist, and working fine 
![ff](https://user-images.githubusercontent.com/21218053/76500042-816c4200-6448-11ea-862c-18f599b69540.png)

- full reset doesn't clean the output docsify directory,  create test.md file in any of docsite directory sub-folders , e.g info_grid. Then do `jsx wiki-load -r` and you will notice that the created file was deleted.
